gmcs/validate.py

```diff
@@ -1758,13 +1758,30 @@ def validate_adnominal_possession(ch, vr):
                 if not inst.get('orth'):
                     mess = 'You must give the spelling for this pronoun.'
                     vr.err(inst.full_key+'_orth', mess)
+
+                # These variables are used to test if there are duplicated features
+                # in poss-pron_instance_feat path
+                features_map = {}
+                index = 1
```

Comment:
I would add a comment explaining what the role of `index` is versus `count`

Reply:
Hi Olga, thank you for your suggestions! Please check my updated comment.